src/mixins/java/org/spongepowered/common/mixin/tracker/world/chunk/ChunkMixin_Tracker.java

```diff
@@ -421,24 +439,25 @@ public BlockState setBlockState(final BlockPos pos, final BlockState state, fina
             this.tileEntities.get(pos).remove();
         }
         added.validate();
-        ((ActiveChunkReferantBridge) added).bridge$setActiveChunk((ChunkBridge) this);
+        ((ActiveChunkReferantBridge) added).bridge$setActiveChunk(this);
         this.tileEntities.put(pos, added);
     }
 
     @Inject(
             method = "addTileEntity(Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/tileentity/TileEntity;)V",
             at = @At(value = "INVOKE", target = "Lnet/minecraft/tileentity/TileEntity;validate()V"))
     private void tracker$SetActiveChunkOnTileEntityAdd(final BlockPos pos, final TileEntity tileEntityIn, final CallbackInfo ci) {
-        ((ActiveChunkReferantBridge) tileEntityIn).bridge$setActiveChunk((ChunkBridge) this);
+        ((ActiveChunkReferantBridge) tileEntityIn).bridge$setActiveChunk(this);
         // Make sure to set creator/notifier for TE if any chunk data exists
         // Failure to do this during chunk load will cause TE's to not have proper user tracking
-        ((CreatorTrackedBridge) tileEntityIn).tracked$setTrackedUUID(PlayerTracker.Type.CREATOR, ((ChunkBridge) this).bridge$getBlockCreatorUUID(pos).orElse(null));
-        ((CreatorTrackedBridge) tileEntityIn).tracked$setTrackedUUID(PlayerTracker.Type.NOTIFIER, ((ChunkBridge) this).bridge$getBlockNotifierUUID(pos).orElse(null));
+        // TODO - Reimplement player uuid tracking.
+        ((CreatorTrackedBridge) tileEntityIn).tracked$setTrackedUUID(PlayerTracker.Type.CREATOR, null);
+        ((CreatorTrackedBridge) tileEntityIn).tracked$setTrackedUUID(PlayerTracker.Type.NOTIFIER, null);
```

Comment:
@Zidane this is the bit that if you revert the changes I made here, this will cause the client to just hang itself on loading a world in single player.